src/bit_set.rs

```diff
@@ -41,6 +41,17 @@ impl Clone for Bitset {
     }
 }
 
+impl PartialEq for Bitset {
+	#[inline]
+	fn eq(&self, other: &Self) -> bool { 
+		self.head == other.head || match (self.tail.as_ref(), other.tail.as_ref()) {
```

Comment:
shouldn't this be `&&` instead of `||`?

Reply:
You're right! Fixing this, even if I ended up not using it anymore.